index.html

```diff
@@ -1,171 +1,287 @@
-<li class="lists"><a href="#"><img src="./images/Frame 10.png" alt="html"></a>
-<li class="lists"><a href="#"><img src="./images/Frame 11.png" alt="css"></a>
-</ul> 
-         <ul class="get">
-          <li class="language"> Frameworks</li>
-         
-          <li><i class="fa fa-chevron-right"></i></li>
-             <hr class="line">
-         </ul>
-          <ul class="get">
-            <li class="language">Skills</li>
-            
-          <li><i class="fa fa-chevron-right"></i></li>
-             <hr class="line"></ul>
-             
-            
-
-
-            
-        </ul>
-        
-    </ul>
-      </div>  
+            <ul class="language-list">
+              <li class="list">
+                <a href="#"
+                  ><img src="./images/Frame 9.png" alt="javascript"
+                /></a>
+              </li>
+              <li class="lists">
+                <a href="#"><img src="./images/Frame 10.png" alt="html" /></a>
+              </li>
+              <li class="lists">
+                <a href="#"><img src="./images/Frame 11.png" alt="css" /></a>
+              </li>
+            </ul>
+            <ul class="get">
+              <li class="language">Frameworks</li>
+              <li><i class="fa fa-chevron-right"></i></li>
+              <hr class="line" />
+            </ul>
+            <ul class="get">
+              <li class="language">Skills</li>
+              <li><i class="fa fa-chevron-right"></i></li>
+              <hr class="line" />
+            </ul>
+          </ul>
+        </div>
+      </ul>
     </section>
-
-</body>
+    <footer class="contact-form">
+      <div class="contact">Contact me</div>
+      <p class="last-page">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="https://formspree.io/f/mpzeyeee" method="Post">
+        <ul>
+          <li>
+            <input
+              class="first-type"
+              type="text"
+              name="userName"
+              maxlength="30"
+              placeholder="userName"
+              required
+            />
+          </li>
+          <li>
+            <input
+              class="second-type"
+              type="email"
+              name="email"
+              placeholder="example@gmail.com"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              name="msg"
+              id="10"
+              cols="30"
+              rows="6"
+              maxlength="500"
+              placeholder="write your message here"
+              required
+            ></textarea>
+          </li>
+          <li>
+            <input
+              class="btn-submit"
+              type="submit"
+              name="submit"
+              value="Get in touch"
+            />
```

Comment:
- [ ] Great job implementing some of the requirements suggested by the previous reviewer: :+1: :+1:  But I think there are still some improvements you need to make before moving to the next milestone. 
- -  [ ] As pointed out by the previous reviewer, your form is not properly aligned with the rest of the content on your page, You have a space at the right and left of the page. This does not look good in the UI and also doesn't look like the Figma design. As part of the requirement for this milestone is to replicate the given design, I'll suggest you over your styling one more time and adjust this. :+1: :smile: :+1: 

- SCREENSHOT FOR REFERENCE:
![Selection_031](https://user-images.githubusercontent.com/107632780/236046870-e6c9ad47-297d-484a-9b1d-477f1649836c.png)


- [ ] Also as suggested by the reviewer, you did a great job implementing the button interaction, But the button background color on hover is different from what's on your design. I'll also suggest you get the correct color from Figma and update this accordingly. :+1: :smile: :airplane: 

- VIDEO FOR REFERNCE:

https://user-images.githubusercontent.com/107632780/236047254-a65fcf11-4705-409c-9b58-6888e8cc1578.mp4